packages/docs/developer-resources/start.md

```diff
@@ -1,11 +1,30 @@
 # Start
 
-The following examples will serve as a guide to understand the Celo SDK:
+The following examples and guides will serve as a guide to understand building on Celo for various platforms:
```

Comment:
nit: possible reword: "The following examples and guides will help you start building on Celo."